staging/src/k8s.io/apiserver/pkg/util/flowcontrol/fairqueuing/queueset/queueset_test.go

```diff
@@ -746,38 +747,45 @@ func TestTooWide(t *testing.T) {
 
 func TestWindup(t *testing.T) {
 	metrics.Register()
-	now := time.Now()
+	for _, testcase := range []struct {
+		margin2     float64
+		expectFair2 bool
+	}{{margin2: 0.26, expectFair2: true}, {margin2: 0.1, expectFair2: false}} {
```

Comment:
nit: can we define the cases as we mostly try to do across codebase:
```
testCases := []struct {
	name        string
	margin2     float64
	expectFair2 bool
}{
  {name: "fair", margin2: 0.26, expectFair2: true},
  {name: "unfair", margin2: 0.1, expectFair2: false},
}

for _, testcase := range testCases {
```

Reply:
Are you concerned about having a name for the slice of test cases, or putting each case on its own line, or both?  I can imagine logic for the latter (making line-based edits independent), but the former is not obvious to me.